server/pulp/plugins/file/distributor.py

```diff
@@ -200,6 +200,16 @@ def post_repo_publish(self, repo, config):
         """
         pass
 
+    @staticmethod
+    def _target_symlink_path(build_dir, target_path):
+        norm_path = os.path.normpath(target_path)
+        if os.path.isabs(norm_path):
+            raise ValueError('Symlink target path must not be absolute: %s' % target_path)
+        if norm_path == '..' or norm_path.startswith('../'):
```

Comment:
I think `startswith('..')`  will cover both cases.

Reply:
I don't think so: `..afile` is a perfectly valid path (i.e. filename) that starts with `..`

Reply:
An odd case but fair enough.